src/NgGrid.ts

```diff
@@ -64,29 +85,29 @@ export class NgGrid implements OnInit, DoCheck, OnDestroy {
 	private _autoResize: boolean = false;
 	private _differ: KeyValueDiffer;
 	private _destroyed: boolean = false;
-	
+
 	//	Default config
-	private static CONST_DEFAULT_CONFIG = {
-		'margins': [10],
-		'draggable': true,
-		'resizable': true,
-		'max_cols': 0,
-		'max_rows': 0,
-		'visible_cols': 0,
-		'visible_rows': 0,
-		'col_width': 250,
-		'row_height': 250,
-		'cascade': 'up',
-		'min_width': 100,
-		'min_height': 100,
-		'fix_to_grid': false,
-		'auto_style': true,
-		'auto_resize': false
+	private static CONST_DEFAULT_CONFIG = <NgGridConfig>{

```

Comment:
Is there a reason for this over `private static CONST_DEFAULT_CONFIG: NgGridConfig = {` ?